contracts/package.json

```diff
@@ -76,7 +77,8 @@
   "dependencies": {
     "@eth-optimism/contracts": "^0.5.21",
     "@openzeppelin/contracts": "~4.3.3",
-    "@openzeppelin/contracts-upgradeable": "^4.7.3",
+    "@openzeppelin/contracts-upgradeable-4.7.3": "npm:@openzeppelin/contracts-upgradeable@v4.7.3",
+    "@openzeppelin/contracts-upgradeable": "^4.8.1",
```

Comment:
Can't we use 4.7.3? Is there anything in 4.8.1 that you ultimately need?

Reply:
There have been a couple of actual solidity code changes since `4.7.3`, but doesn't seem like anything vital.
See https://github.com/OpenZeppelin/openzeppelin-contracts/commits/v4.8.1/contracts/proxy/utils/Initializable.sol , everything above "Merge v4.7 back into master".

The reason I used `4.8.1` was more to stay in sync with the latest version that `@openzeppelin/hardhat-upgrades` uses as a peer dependency.

I'm open to us pinning to older versions - what is the argument for doing that? Simplicity in the repo dependencies?

Reply:
I would prefer to keep one of these versions for clarity/simplicity. It looks very odd to keep both versions given they are the same major versions. Have you considered removing 4.7.3?

Reply:
That would force a code change within the VRF contracts that are dependent on 4.7.3.

I tried moving both to local `vendor` contracts, but there is something hardhat doesn't like about that when compiling the artifacts.